syntax = "proto3";

package transcription;

import "google/protobuf/timestamp.proto";

message TranscribeRequest {
    string audio = 1;
    bool is_long_audio = 2;
    string language = 3;
}

message TranscribeResponse {
    string transcript = 1;
    google.protobuf.Timestamp timestamp = 2;
}

service TranscriptionService {
    rpc Transcribe (TranscribeRequest) returns (TranscribeResponse);
}